### PR TITLE
Add WIP newsletter

### PR DESCRIPTION
**Pull request by jtoar**
This is an example of a more-legit newsletter. I'm adding it to the website because 1) I used the website's styling 2) if we go ahead with a newsletter like this, maybe we'd add them to the website anyway?

But the upshot of this PR is: there's a new section in the nav (newsletter) with the more-legit newsletter:

newsletter

One for-sure todo is fixing the formatting in "on this page" aside.

### CODE REVIEW

**Comment by peterp** [comment]
This is awesome!

File: lib/build.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const octokit = Octokit({
 })
 
 const { create: createDocs, buildNav } = require('./docutron.js')
-const { publish: publishSearch, getObjectIDs } = require('./search.js')
+// const { publish: publishSearch, getObjectIDs } = require('./search.js')
```
Comment (cannikin) [inline lib/build.js:13]: Make sure this doesn't become part of the real PR! 😬

File: lib/build.js [hunk 2]
```diff
@@ -145,7 +151,7 @@ const setPageVersions = (version) => {
 }
 
 const main = async () => {
-  const objectIDs = await getObjectIDs()
+  // const objectIDs = await getObjectIDs()
```
Comment (cannikin) [inline lib/build.js:154]: Likewise

File: lib/build.js [hunk 3]
```diff
@@ -162,7 +168,7 @@ const main = async () => {
       const pages = createDocs(markdown, section.name, { ...file, version })
 
       navLinks += buildNav(pages, section.name, index).join('\n')
-      await publishSearch(markdown, section.name, { ...file, objectIDs })
+      // await publishSearch(markdown, section.name, { ...file, objectIDs })
```
Comment (cannikin) [inline lib/build.js:171]: Likewise x2

**Comment by cannikin** [comment]
I'm thinking we'll want to exclude the Newsletter from search, otherwise they'll start dominating the search results. Maybe we add some kind of `searchable` key to the big `SECTIONS` object, which is `true` for everything there now, but `false` for the newsletter section. Then check for that when compiling the sections for search.

File: lib/build.js (hunk 1 above)
Comment (jtoar) [reply lib/build.js:13]: Ah--super sorry! (x3)

File: lib/build.js [hunk 4]
```diff
@@ -162,7 +180,7 @@ const main = async () => {
       const pages = createDocs(markdown, section.name, { ...file, version })
 
       navLinks += buildNav(pages, section.name, index).join('\n')
-      await publishSearch(markdown, section.name, { ...file, objectIDs })
+      if (section.searchable) await publishSearch(markdown, section.name, { ...file, objectIDs })
```
Comment (jtoar) [inline lib/build.js:183]: @cannikin Is something like this what you meant?

Comment (jtoar) [inline lib/build.js:183]: @cannikin Is something like this what you meant? (Sorry I resolved the first one, didn't know what I was doing add-single-comment wise.)

Comment (cannikin) [reply lib/build.js:183]: Yup! Love it.

**Comment by jtoar** [comment]
Going to close this one; served its purpose on the road to newsletter v0.0.1.

**Pull request closed without merging**